packages/core/src/queries/custom-phrase.ts

```diff
@@ -8,6 +8,16 @@ import { DeletionError } from '@/errors/SlonikError';
 
 const { table, fields } = convertToIdentifiers(CustomPhrases);
 
+export const findAllCustomLanguageKeys = async () => {
+  const rows = await envSet.pool.many<{ languageKey: string }>(sql`
```

Comment:
will `many` throw an error if there is no record?

Reply:
refactored and locally tested. PTAL.

```ts
export const findAllCustomLanguageKeys = async () => {
  const rows = await manyRows<{ languageKey: string }>(
    envSet.pool.query(sql`
      select ${fields.languageKey}
      from ${table}
      order by ${fields.languageKey}
    `)
  );

  return rows.map((row) => row.languageKey);
};
```